.changeset/strong-mice-talk.md

```diff
@@ -0,0 +1,8 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+
+Add a function `isRepoSuperset` to determines whether the `view` schema allows a superset of the documents that the `stored` schema allows.
```

Comment:
Since we're not exporting this, I feel like more generic language would be better, or at least add a disclaimer to the changeset along the lines of "changes are not customer-facing and make progress toward future plans in Tree's schema evolution space."